Edit UG (#301)
AY2324S1-CS2103T-T17-2/tp: docs/UserGuide.md

**:information_source: Notes:**<br>
* The `find` command searches only the company name.
* Companies with names that contain **any** of the keywords will be returned.
* Keyword is case-insensitive (e.g. `tiktok` will match `TikTok`).
* The order of your keywords doesn't matter (e.g. `tiktok google` will match `google tiktok`).

> * LinkMeIn's find command will always find from the full list of companies. This means that if you have previously done `filter s/pa`, doing find next, will not find from the filtered list of companies.

Reviewer: ```suggestion
* LinkMeIn's find command will always find from the **full** list of companies. This means that if you have previously used `filter`, performing `find` will not search from the filtered list of companies.
```

> * LinkMeIn's `find` command will always find from the **full** list of companies. This means that if you have previously used `filter s/pa`, performing `find`, will not search from the filtered list of companies.

Reviewer: is there a need to specify an example?

> * LinkMeIn's `find` command will always find from the **full** list of companies. This means that if you have previously used `filter s/pa`, performing `find`, will not search from the filtered list of companies.

Reviewer: Is there a need to specify an example? Perhaps just saying previously used `filter` might be better

Author: true ok removed the example

<div markdown="block" class="alert alert-info">
**:information_source: Note:**<br>

> * Upon launching LinkMeIn, you will see your full list of companies. (i.e. list command is executed automatically).

Reviewer 2: ```suggestion
* Upon launching LinkMeIn, you will see your full list of companies. (i.e. `list` command is executed automatically).
```

> [**APPLICATION_STATUS**](#application-status-parameter-description) should be one of the following: `PA`, `PI`, `PO`, `A`, `R`.

<div markdown="block" class="alert alert-info">
**:information_source: Note:**<br>

> * LinkMeIn's filter command will always filter from the full list of companies. This means that if you have previously done `find ds`, doing filter next, will not filter from the list of companies that you have searched for.

Reviewer 2: ```suggestion
* LinkMeIn's `filter` command will always filter from the full list of companies. This means that if you have previously done `find ds`, doing filter next, will not filter from the list of companies that you have searched for.
```

> * LinkMeIn's filter command will always filter from the full list of companies. This means that if you have previously done `find ds`, doing filter next, will not filter from the list of companies that you have searched for.

Reviewer: ```suggestion
* LinkMeIn's filter command will always filter from the **full** list of companies. This means that if you have previously used `find`, performing `filter`, will not filter from the list of companies that you have searched for.
```

> * LinkMeIn's `filter` command will always filter from the **full** list of companies. This means that if you have previously used `find ds`, performing `filter`, will not filter from the list of companies that you have searched for.

Reviewer: same as before

Author: removed too

* The case of `SORT_ORDER` does not matter (e.g. `ASCENDING` and `ascending` are treated the same).
</div>

<div markdown="block" class="alert alert-success">
**:bulb: Useful Tip:**<br>

> * If no `SORT_ORDER` is specified, it will default to `ascending`. Use this to quickly see which deadlines are 

Reviewer 2: ```suggestion
* If no `SORT_ORDER` is specified, it will be set to `ascending` by default. Use this to quickly see which deadlines are 
```

| **CLI**              | Command-Line Interface (CLI) is a text-based user interface where users interact with the application by typing commands.                                                                                                           |
| **Case-insensitive** | Upper case and lower case are being treated the same way. For example, `tiktok` and `TikTok` will be considered the same.                                                                                                           |
| **Command**          | A command is an instruction given by a user to LinkMeIn to perform a specific action. For example,`add` command is a command to add the company's application into LinkMeIn.                                                        |
| **GUI**              | Graphical User Interface (GUI) is a visual method to interact with software using icons, buttons, and windows. GUI provides a user-friendly way to interact with software using graphical elements rather than text-based commands. |

> | **Index**            | Refers to the index number shown in the displayed company list                                                                                                                                                                      |

Reviewer 2: missing full stop

| **JSON**             | JSON stands for JavaScript Object Notation. It is lightweight format for data interchange, easy to read and write for humans, and easy to parse for machines. Often used in web applications and configuration files.               |
| **Parameter**        | Parameter is similar to a field in a form you have to fill up. For example, in the command `edit 1 c/COMPANY_NAME e/EMAIL`, `COMPANY_NAME` and `EMAIL` are parameters in the command.                                               |
| **Prefix**           | Prefix is a keyword that is used to identify the parameter. For example, in the command `edit 1 c/COMPANY_NAME e/EMAIL`, `c/` and `e/` are prefixes.                                                                                |

> | **Case-insensitive** | Upper case and lower case are being treated the same way. For example, `tiktok` and `TikTok` will be considered the same.                                                                                                           |

Reviewer: Arrange glossary in alphabetical order

| **Index**            | Refers to the index number shown in the displayed company list.                                                                                                                                                                      |
| **JAR**              | JAR stands for Java Archive and is a package file format typically used to aggregate many Java class files and associated metadata and resources into one file for distribution.                                                    |
| **JSON**             | JSON stands for JavaScript Object Notation. It is lightweight format for data interchange, easy to read and write for humans, and easy to parse for machines. Often used in web applications and configuration files.               |
| **Parameter**        | Parameter is similar to a field in a form you have to fill up. For example, in the command `edit 1 c/COMPANY_NAME e/EMAIL`, `COMPANY_NAME` and `EMAIL` are parameters in the command.                                               |

> | **Prefix**           | Prefix is a keyword that is used to identify the parameter. For example, in the command `edit 1 c/COMPANY_NAME e/EMAIL`, `c/` and `e/` are prefixes.                                                                                |

Reviewer: Command and other terms are repeated. Not in alphabetical order also
